models/staging/__sources.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: ecom
-    schema: "{{target.schema}}_raw"
+    schema: "{{ var('jaffle_raw_schema') }}"
```

Comment:
Previously this would have evaluated to 'analytics_raw' (evaluated from "{{target.schema}}_raw").

Since the files are source with the meta->external_location instruction, this shouldn't cause any breakage.